src/stancjs/stancjs.ml

```diff
@@ -45,21 +45,20 @@ let stan2cpp model_name model_string flags =
           "Semantic check failed but reported no errors. This should never \
            happen."
   in
-  if is_flag_set "version" then Result.Ok (Fmt.strf "%s" version, [])
```

Comment:
I think we want to do something to keep the ability for phases of the compiler to return warnings rather than delete these for-now empty lists.